src/core/services/gesture/gesture.js

```diff
@@ -302,7 +302,7 @@ angular.module('material.core')
     onCancel: angular.noop,
     options: {},
 
-    dispatchEvent: dispatchEvent,
+    dispatchEvent: typeof jQuery !== 'undefined' && angular.element === jQuery ? jQueryDispatchEvent : nativeDispatchEvent,
```

Comment:
Beware that this will probably break once https://github.com/angular/angular.js/pull/10761 lands (and I believe it's pretty close to landing).